_posts/2021-03-14-curating/curating.Rmd

```diff
@@ -0,0 +1,329 @@
+    embed2 = purrr::map(url, include_tweet),
+    embed3 = purrr::map(embed2, ~ .x[[2]]),
+    img_source = glue::glue('<img src="{full_path}tweet1.png">')
+    ) 
+```
+
+
+```{r}
+#tooltip_css <- "background-color:gray;color:white;padding:10px;border-radius:5px;"
+```
+
+```{r}
+poll_colors <- c(
+  `Poll #1` = "#fae1dd",
+  `Poll #2` = "#ddfae1",
+  `Poll #3` = "#e1ddfa"
+)
+```
+
+
+
+```{r}
+p <- polls %>% 
+  # embed is html copied and pasted from twitter ---
+  # embed3 comes from tweetrmd package, looks same as embed ----
+  # img_source is image taken from tweetrmd, couldn't get this to work ----
+  ggplot(aes(x = 1, y = tweet_label, fill = poll_label, tooltip = embed3, onclick = content_click)) +
+  facet_wrap(. ~ day, ncol = 6, drop = F) +
+  geom_tile_interactive(color = "white") +
+  geom_text(aes(label = content_label), size = 2.5) +
+  theme_minimal() +
+  scale_fill_manual(values = poll_colors) +
+  theme(
+    # removes grid lines from plot ----
+    panel.grid = element_blank(),
+    # moves legend to top instead of side ----
+    legend.position = "top",
+    # removes title from legend, often superfluous ----
+    legend.title = element_blank(),
+    # creates the light gray box around the plot, may or may not want this ----
+    #panel.background = element_rect(color = "#F2F2F2"),
+    # creates the gray background boxes for faceting labels ----
+    strip.background = element_rect(
+      color = "#F2F2F2",
+      fill = "#F2F2F2"
+    ),
+    # this produces a fully left justified title ----
+    plot.title.position = "plot",
+    axis.text.x = element_blank(),
+    axis.title.x = element_blank(),
+    #axis.text.y = element_blank(),
+    axis.title.y = element_blank(),
+    )
+```
+
+
+```{r}
+girafe(ggobj = p,
+       #opts_tooltip(use_fill = TRUE)
+       #opts_tooltip(css = tooltip_css, use_fill = TRUE)
+       )
+```
+
+
+
+# Styling content
+
+Additionally, I considered how to visually style content beyond text in a tweet. I asked on [R-Ladies slack](https://rladies-community-slack.herokuapp.com/){target="_blank"} about R-Ladies styles, and I was directed to the xaringan R-Ladies [css](https://github.com/yihui/xaringan/tree/master/inst/rmarkdown/templates/xaringan/resources){target="_blank"} and the R-Ladies [branding](https://guide.rladies.org/organization/tech/brand/){target="_blank"} guides. You are not required to use R-Ladies style and branding, but it was convenient for me. 
+
+I developed two visual layouts using the Google slide [template](https://docs.google.com/presentation/d/1sriC2biLPYza_TtGiZkrNDsv3AMg6dvhqw2yit4wNnA/edit){target="_blank"} from R-Ladies branding (see tweets for [blogdown vs dstill](https://twitter.com/WeAreRLadies/status/1363144545677017089){target="_blank"} and [asking for help online](https://twitter.com/WeAreRLadies/status/1362370580708790274){target="_blank"}).
+
+```{r echo=FALSE, fig.cap="Comparison of blogdown vs distill styled using R-Ladies Google slide template."}
```

Comment:
also add fig.alt with more details about the text?

Reply:
good catch!